shell/platform/common/client_wrapper/include/flutter/event_channel.h

```diff
@@ -106,7 +106,7 @@ class EventChannel {
             if (error) {
               result = codec->EncodeErrorEnvelope(error->error_code,
                                                   error->error_message,
-                                                  error->error_details);
+                                                  error->error_details.get());
```

Comment:
This method points to an abstract method that we implement in the standard and json codecs, both of which copy the value pointed to by the parameter.